src/pages/keys/KeysContent.tsx

```
@@ -785,6 +873,110 @@ export default function KeysContent() {
+                              ) {
+                                return;
+                              }
+                              handleAddAppId();
+                            }
+                          }}
+                          className="flex-1 px-3 py-2.5 border border-gray-300 rounded-md focus:outline-none focus:ring-2 focus:ring-[#de3624] focus:border-transparent transition-all duration-150"
+                        />
+                        <button
+                          onClick={handleAddAppId}
+                          className="px-4 py-2.5 bg-[#df3326] text-white font-medium rounded-md hover:bg-[#c72a1f] active:scale-[0.98] transition-all duration-150"
+                        >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)